client/app/models/project.js

```diff
@@ -125,6 +126,11 @@ export default class ProjectModel extends Model {
 
   @attr() dcpNoticeddate;
 
+  @computed('formattedNoticedDate')
+  get formattedNoticedDate() {
+    return moment(this.dcpNoticeddate).utcOffset(300).format('MM/D/YYYY');
```

Comment:
I believe that NYC is not always UTC -5, that it changes with daylight savings.  The docs you linked indicate that a more appropriate use in this case may be moment-timezone.

Also, why have leading zeroes on the month but not the day?  The site seems to use 'M/D/YYYY' pretty consistently.

Reply:
@dhochbaum-dcp  great points. I thought it wouldn't matter since the only thing we were using the `dcpNoticeddate` was to display to users a date, since the difference or 4 or 5 hours would still get us the correct date. I refactored the computed method to check whether or not it is DST and adjust the appropriate offset amount. I tried to use `moment.tz(...)` but that wound up constantly breaking the app.